packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.5.1
+* Added windowName web only parameter
```

Comment:
Please leave a space between the version and the list. Also, let's call the parameter:

`webOnlyWindowName`

(The parameter is used as the "windowName" of the `window.open` function, so I think that's the best name for now!)